webpack/JobWizard/JobWizard.js

```diff
@@ -187,18 +191,80 @@ export const JobWizard = () => {
     },
     {
       name: WIZARD_TITLES.schedule,
-      component: (
-        <Schedule
-          scheduleValue={scheduleValue}
-          setScheduleValue={setScheduleValue}
-          setValid={newValue => {
-            setValid(currentValid => ({ ...currentValid, schedule: newValue }));
-          }}
-        />
-      ),
       canJumpTo: isTemplate && valid.hostsAndInputs && valid.advanced,
       enableNext:
         isTemplate && valid.hostsAndInputs && valid.advanced && valid.schedule,
+      steps: [
+        {
+          name: WIZARD_TITLES.typeOfExecution,
+          component: (
+            <ScheduleType
+              scheduleType={scheduleValue.scheduleType}
+              isTypeStatic={scheduleValue.isTypeStatic}
+              setScheduleValue={setScheduleValue}
+              setValid={newValue => {
+                setValid(currentValid => ({
+                  ...currentValid,
+                  schedule: newValue,
+                }));
+              }}
+            />
+          ),
+          canJumpTo: isTemplate && valid.hostsAndInputs && valid.advanced,
+
+          enableNext: isTemplate && valid.hostsAndInputs && valid.advanced,
+        },
+        ...(scheduleValue.scheduleType === SCHEDULE_TYPES.FUTURE
+          ? [
+              {
+                name: SCHEDULE_TYPES.FUTURE,
+                component: (
+                  <ScheduleFuture
+                    scheduleValue={scheduleValue}
+                    setScheduleValue={setScheduleValue}
+                    setValid={newValue => {
+                      setValid(currentValid => ({
+                        ...currentValid,
+                        schedule: newValue,
+                      }));
+                    }}
+                  />
+                ),
+                canJumpTo: isTemplate && valid.hostsAndInputs && valid.advanced,
+                enableNext:
+                  isTemplate &&
+                  valid.hostsAndInputs &&
+                  valid.advanced &&
+                  valid.schedule,
+              },
+            ]
+          : []),
+        ...(scheduleValue.scheduleType === SCHEDULE_TYPES.RECCURING
```

Comment:
```suggestion
        ...(scheduleValue.scheduleType === SCHEDULE_TYPES.RECURRING
```

Recurring has two R's, not two C's.. should be changed everywhere ;)